releasenotes/notes/enable-ha-supported-integrations-32c59c3bb1b8caf5.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Enables High Availability on SNMP, Cisco SD-WAN, and Network Path integrations.
```

Comment:
I believe these checks were already supported by HA. Would it make sense to update this to match the PR description?

Reply:
Good idea, updated here https://github.com/DataDog/datadog-agent/pull/34158/commits/4584bf44c66eed26a90af8fdc9eeacff59e6c973